crates/aiken-project/src/tests/gen_uplc.rs

```diff
@@ -5801,9 +5801,9 @@ fn opaque_value_in_datum() {
             .lambda("ctx")
             .lambda("red")
             .lambda("dat")
+            .constr_fields_exposer()
             .lambda("expect[(_,amount)]=val.inner.inner")
             .apply(Term::string("expect [(_, amount)] = val.inner.inner"))
-            .constr_fields_exposer()
```

Comment:
Funny enough it happens because we traverse forwards over expressions over and previously we traversed backwards which made us encounter the expect first vs the record access.

Reply:
So it would be a pain to try to continue going backward. So now we just go forwards and encounter the expressions in forward order.